integration_testing/features/super-admin/super-admin-update-channel-import-more.feature

```diff
@@ -1,88 +1,75 @@
-      And I see the total number and size of <channel> channel resources
-      And I see N resources from <channel> channel are listed as *On your device*
-      And I see the *Import* button is inactive 
-    When I check the previously not imported <topic> topic checkbox
-    Then I see the *Import* button is active 
-      And I see the *N resources selected* flag for the <topic> topic
-      And I see the values for *Content selected* increase
-    When I click the *Import* button
-    Then I see *Device > Channels* page again
-      And I see the *Importing content...* label and blue progress bar with the percentage increasing 
-    When the import process concludes
-    Then I see the progress bar at 100%
-      And I see the *Finished! Click "Close" button to see changes.* flag
-      And I see the *Close* button is active
-    When I click *Close* 
-    Then I see the *Channels* page is reloaded 
-    When I go to *Learn > Channels* page
-      And I select the <channel> channel
-    Then I see the <topic> topic
+      When I click *Manage* button for the <channel> channel
+      Then I see the *Loading...* in the top bar
+        And I see the "Generating channel listing. This could take a few minutes..." notification
+      When the channel listing is generated
+      Then I am on the *Manage '<channel>'* page
+        And I see the *Version N is available* blue notification
+        And I see the *View changes* button
+        And I see the channel page with logo, name, and the current version on the device
+        And I see the total number and size of <channel> channel resources 
+        # these values need to refer to the current version on the device
+        And I see M resources from <channel> channel are listed as *On your device*
+        And I see the list of topics for the <channel> channel
+        And I see the *Import* button is inactive
+
+  Scenario: Review changes in the new channel version
+    When I click the *View changes* button
+    Then I see the *'<channel>' > Version N of '<channel>' is available* page
+      And I see list of changes of the resources (new, to be deleted, to be updated) if I choose to update the channel
+      And I see the *Update channel* button
+      And I see the information about the changes in previous versions of the channel # if stated on Studio during publishing
+
+  Scenario: Update channel to the new version
+    When I click the *Update channel* button
+    Then I see *Update channel* modal asking for confirmation
+    When I click *Continue* button
+    Then I see *Device > Task manager* page with the *Update '<channel>' to version N* task in progress
+      And I see the green progress bar with the percentage increasing 
+      And I see the number and size of the resources being updated
+      And the number of resources needs to be close/identical to the *Resources to be updated* value in the previous scenario
```

Comment:
@jonboiser @indirectlylit Is this the correct assumption? Is it possible to state the range of _discrepancies_?